modules/client/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ TEMPLATE:
 - Change 1, 2, 3
 
 -->
+## UPCOMING
+### Changed
+- `proposalId` is now a `string` everywhere and follows the new general format
```

Comment:
Could we also add the mention to `canExecute()` and `execute()` now receiving the `proposalId` directly?

Reply:
done